### PR TITLE
[integrations/azure] Update az cli commands

### LINKED ISSUE #2780: Invalid parameters when creating an Azure AD application
**Describe the issue**

Some parameters are incorrect. See below:

```bash
# Create the app
APP_ID=$(az ad app create \
  --display-name "${APP_NAME}" \
  --available-to-other-tenants false \
  --oauth2-allow-implicit-flow false \
  | jq -r .appId)
```
When I run this script, it reported `--available-to-other-tenants` and `oauth2-allow-implicit-flow` cannot be recognized. Checked the doc https://docs.microsoft.com/en-us/cli/azure/ad/app?view=azure-cli-latest#az-ad-app-create, there are no such two parameters.

```bash
az ad app credential reset \
  --id "${APP_ID}" \
  --years 2 \
  --password $(openssl rand -base64 30)
```

For the above script, `--password` cannot be recognized. Checked the doc https://docs.microsoft.com/en-us/cli/azure/ad/app/credential?view=azure-cli-latest#az-ad-app-credential-reset. There is no such parameter.

```bash
SERVICE_PRINCIPAL_ID=$(az ad sp create \
  --id "${APP_ID}" \
  | jq -r .objectId)
echo "Service Principal ID: ${SERVICE_PRINCIPAL_ID}"
```

For the above script, the content does not have a field called `objectId` so the output is null. I think it has been renamed as `id`.

**URL of the docs**

https://docs.dapr.io/developing-applications/integrations/azure/authenticating-azure/#creating-an-azure-ad-application
https://docs.microsoft.com/en-us/cli/azure/ad/app/credential?view=azure-cli-latest#az-ad-app-credential-reset

The output is also incorrect. I think Azure CLI has been updated.

**Expected content**

The scripts should run correctly.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by hhunter-ms**
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

## Description

Update outdated az cli commands for:
- az ad app create
- az ad app credential reset

Update jq command for:
- az ad sp create

Update outputs

## Issue reference

PR will close: #2780

### CODE REVIEW

**Comment by hhunter-ms** [comment]
assigning @ItalyPaleAle for review - based on examples in the doc, it looks like you were the original author and may have better knowledge of az cli commands!

File: daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md [hunk 1]
```diff
@@ -110,8 +110,8 @@ APP_NAME="dapr-application"
 # Create the app
 APP_ID=$(az ad app create \
   --display-name "${APP_NAME}" \
-  --available-to-other-tenants false \
-  --oauth2-allow-implicit-flow false \
+  --enable-access-token-issuance false \
+  --enable-id-token-issuance false
```
Comment (ItalyPaleAle) [inline daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md]: These two flags seem pretty different and I don't think we should use them. I'd just remove the flags if they've been deleted.

**Comment by ItalyPaleAle** [comment]
@hhunter-ms thanks for this. Looks like the commands have changed since I wrote the docs originally... I think they _look_ ok besides the change I requested, but I haven't tested these.

File: daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md (hunk 1 above)
Comment (hhunter-ms) [reply daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md]: @ItalyPaleAle Both the `--enable....` parameters look like they would replace `--oauth2-allow-implicit-flow`, per [`az ad app`](https://docs.microsoft.com/en-us/cli/azure/ad/app?view=azure-cli-latest#az-ad-app-create-optional-parameters) reference... but I can remove if they aren't necessary for the example. What do you think?

Comment (hhunter-ms) [reply daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md]: I couldn't find anything to replace `--available-to-other-tenants`

**Comment by hhunter-ms** [comment]
@ItalyPaleAle asked a quick q in the review above - thank you! I tested the example with the new commands and it worked for me!

File: daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md (hunk 1 above)
Comment (ItalyPaleAle) [reply daprdocs/content/en/developing-applications/integrations/Azure/authenticating-azure.md]: I think they're not needed actually, it should be ok without those flags.

File: daprdocs/content/en/reference/components-reference/supported-bindings/s3.md [hunk 2]
```diff
@@ -75,8 +75,9 @@ When running the Dapr sidecar (daprd) with your application on EKS (AWS Kubernet
 
 ### Using with Minio
 
+<!--IGNORE_LINKS-->
 [Minio](https://min.io/) is a service that exposes local storage as S3-compatible block storage, and it's a popular alternative to S3 especially in development environments. You can use the S3 binding with Minio too, with some configuration tweaks:
```
Comment (greenie-msft) [inline daprdocs/content/en/reference/components-reference/supported-bindings/s3.md:78]: This links seems to work as of now. Can we please try to remove the ignore links and see if it passes?

**Review by greenie-msft — APPROVED** [review]

**Pull request merged**